modules/apps/opensocial/opensocial-portlet/docroot/WEB-INF/src/com/liferay/opensocial/shindig/service/LiferayPersonService.java

```diff
@@ -281,9 +281,7 @@ protected Person getUserPerson(
 
 		sb.append(securityToken.getDomain());
 		sb.append(PortalUtil.getPathImage());
-		sb.append("/user_");
-		sb.append(user.isFemale() ? "female" : "male");
-		sb.append("_portrait?img_id=");
+		sb.append("/user_male_portrait?img_id=");
```

Comment:
Could  this be renamed to `user_portrait`?

Reply:
Fully agreed. 

This part is unfinished and @drewbrokke will continue working on it.

Reply:
@mdelapenya that's the plan. See https://github.com/brianchandotcom/liferay-portal/pull/66186/commits/e22aebaf16ff25446d4858ab6635d32a572e673f